kubernetes/salt/config/master.d/salt-master.conf

```diff
@@ -5,11 +5,11 @@ interface: 0.0.0.0
 event_return: mysql
 
 mysql:
-  host: '127.0.0.1'
+  # salt does not support specifying the UNIX socket location here - as a workaround,
```

Comment:
https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.mysql.html says you can set `mysql.unix_socket: '<path_to_socket_here>'`. Didn't this work?